src/main/resources/db/changelog/changes/05-create-users-roles-table.yaml

```
@@ -0,0 +1,26 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-users-roles-table
+      author: dmytro

```

```suggestion
      author: dmytro_potapov
```
it's better to provide full name as author of changeSet